x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -153,17 +160,27 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
     <Fragment>
       <EuiPage style={{ backgroundColor: 'inherit' }} data-test-subj={`mlPageJobWizard ${jobType}`}>
         <EuiPageBody>
-          <EuiPageContentBody>
-            <Wizard
-              jobCreator={jobCreator}
-              chartLoader={chartLoader}
-              resultsLoader={resultsLoader}
-              chartInterval={chartInterval}
-              jobValidator={jobValidator}
-              existingJobsAndGroups={existingJobsAndGroups}
-              firstWizardStep={firstWizardStep}
-            />
-          </EuiPageContentBody>
+          <EuiPageContent>
+            <EuiPageContentHeader>
+              <EuiPageContentHeaderSection>
+                <EuiTitle>
+                  <h2>Create job: {jobType} </h2>
```

Comment:
Is there a way to easily grab the job type name here, instead of the job type variable?

Reply:
there is not human readable version of the job type in the job creator class, but you could add a function like this to `new_job/common/job_creator/util/general.ts`
```
export function getJobCreatorTitle(jobCreator: JobCreatorType) {
  switch (jobCreator.type) {
    case JOB_TYPE.SINGLE_METRIC:
      return i18n.translate('xpack.ml.newJob.wizard.jobCreatorTitle.singleMetric', {
        defaultMessage: 'Single metric',
      });
    case JOB_TYPE.MULTI_METRIC:
      return i18n.translate('xpack.ml.newJob.wizard.jobCreatorTitle.multiMetric', {
        defaultMessage: 'Multi metric',
      });
    case JOB_TYPE.POPULATION:
      return i18n.translate('xpack.ml.newJob.wizard.jobCreatorTitle.population', {
        defaultMessage: 'Population',
      });
    case JOB_TYPE.ADVANCED:
      return i18n.translate('xpack.ml.newJob.wizard.jobCreatorTitle.advanced', {
        defaultMessage: 'Advanced',
      });
    // case JOB_TYPE.CATEGORIZATION:
    //   return i18n.translate('xpack.ml.newJob.wizard.jobCreatorTitle.categorization', {
    //     defaultMessage: 'Categorization',
    //   });
    default:
      return '';
  }
}
```

Reply:
This is added in.